app/src/main/java/com/example/tour_a_versity_v3/TucPhotos.kt

```diff
@@ -3,9 +3,9 @@ package com.example.tour_a_versity_v3
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 
-class TUC_Photos : AppCompatActivity() {
+class TucPhotos : AppCompatActivity() {
```

Comment:
Changed the name of this class to match the naming conventions for classes as described here:
https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules